contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -167,15 +182,58 @@ def _execute(self, expected_count: int = None, expected_names: List[str or Itera
         if expected_names and not allow_additional_tracks:
             self.expect_eq(names_found, expected_count, "No additional tracks are found")
 
-    @staticmethod
-    def _match(expected_name: str or Iterable[str], found_name: str) -> bool:
-        if isinstance(expected_name, str):
-            return fnmatch(found_name, expected_name)
-        else:
-            for option in expected_name:
-                if fnmatch(found_name, option):
-                    return True
-        return False
+
+class CollapsingTrackBase(MatchTracksBase):
```

Comment:
I don't think this is a good use of inheritance. As far as I can see you only want the `_match` method?  Probably you don't need that, see my comment below, but even if you do, I'd suggest to either make this a free function or introduce it in the `CaptureWindowE2ETestCaseBase`.

Reply:
Agree. I wasn't sure about the best way to do so in python. However, this is obsolete with using the find_tracks method with a filter.